docs/kqueen.rst

```diff
@@ -442,6 +442,42 @@ Provision a Kubernetes cluster using Google Kubernetes Engine
 #. Click ``Submit``.
 #. To track the cluster status, navigate to the KQueen main dashboard.
 
+Provision a Kubernetes cluster using Openstack Kubespray Engine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Log in to Openstack Horizon Dashboard.
+#. Select your Project.
+#. Navigate to Project`-> ``Compute`` -> ``Access & Security`` -> ``API Access`` tab and click
+   ``Download OpenStack RC File``.
+#. Log in to the KQueen web UI.
+#. From the ``Create Provisioner`` page, select ``Openstack Kubespray Engine``.
+#. Specify provisioner requirements with the Openstack RC File, downloaded earlier.
+#. Click ``Deploy Cluster``.
+#. Select the defined Openstack provisioner.
+#. Specify the cluster requirements, and pay attention on the following cases:
+
+   #. ``Ssh key name`` is name of SSH key pair. You should choose from existing pairs or create a new one.
+   #. ``Flavor`` must be at least ``m1.small`` (2GB RAM, dual-core CPU).
```

Comment:
why did you remove information about images? I think it is needed

Reply:
seems no, kubespray works fine with non-customized images, but it works for all normal apps)

Reply:
ok, you are describing ssh since it is presented and required, same applies for an image. why you are listed it here? I think we can describe all the parameters here

Reply:
imo we need to describe parameters, that may cause misunderstandings